sass/partials/components/_article.scss

```diff
@@ -0,0 +1,3 @@
+.article {
```

Comment:
what's the purpose of the bottom margin?  is it truly for all things with class article (full page posts, teasers, etc.) or just teasers?  if just the latter, should we make it more specific?